packages/contracts-bedrock/src/L2/interfaces/IL1Block.sol

```diff
@@ -34,8 +34,11 @@ interface IL1Block {
     )
         external;
     function setL1BlockValuesEcotone() external;
+    function setL1BlockValuesHolocene() external;
     function timestamp() external view returns (uint64);
     function version() external pure returns (string memory);
 
     function __constructor__() external;
+    function eip1559Denominator() external view returns (uint64);
+    function eip1559Elasticity() external view returns (uint64);
```

Comment:
These should likely go above the constructor to follow style

Reply:
done